bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java

```diff
@@ -328,18 +328,22 @@ static String composeMergeConversationFooter(PullRequest pr, Repository localRep
         var commitsLink = commitsLink(pr, base, head);
         String webrevLinks;
         if (webrevs.size() > 0) {
-            var containsConflicts = webrevs.stream().anyMatch(w -> w.type().equals(WebrevDescription.Type.MERGE_CONFLICT));
-            var containsMergeDiffs = webrevs.stream().anyMatch(w -> w.type().equals(WebrevDescription.Type.MERGE_TARGET) ||
-                    w.type().equals(WebrevDescription.Type.MERGE_SOURCE));
-
-            webrevLinks = "The webrev" + (webrevs.size() > 1 ? "s" : "") + " contain" + (webrevs.size() == 1 ? "s" : "") + " " +
-                    (containsConflicts ? "the conflicts with " + pr.targetRef() : "") +
-                    (containsConflicts && containsMergeDiffs ? " and " : "") +
-                    (containsMergeDiffs ? "the adjustments done while merging with regards to each parent branch" : "")
-                    +":\n" +
-                    webrevs.stream()
-                           .map(d -> String.format(" - %s: %s", d.shortLabel(), d.uri()))
-                           .collect(Collectors.joining("\n")) + "\n\n";
+            if (webrevs.stream().noneMatch(w -> w.uri() != null)) {
+                webrevLinks = "Webrev is disabled\n\n";
```

Comment:
I think this should just be `""` or possibly `"\n"`, not sure.

Reply:
```
else {
            webrevLinks = "The merge commit only contains trivial merges, so no merge-specific webrevs have been generated.\n\n";
        }
```
I just tried to align with this.

Reply:
Yes, I figured that, but there is an important difference. That message explains why there is no webrevs where the user would expect them to be, but for a repo where they are completely turned off, I think we shouldn't even mention them.